Add `New Featured Artist: WangleLine` news post (#13503)
ppy/osu-wiki: news/2025/2025-07-13-new-featured-artist-wangleline.md

---
layout: post
title: "New Featured Artist: WangleLine"
date: 2025-07-13 22:00:00 +0000
---

If you haven't heard **WangleLine** yet, your day is about to get a lot better.

![](https://assets.ppy.sh/artists/427/header.jpg)

> German producer **WangleLine** is as prolific as it gets. With 14 albums created to date, plus tracks for the [osu!catch World Cup](https://osu.ppy.sh/wiki/Tournaments/CWC), [Lazer Grand Arena](https://osu.ppy.sh/wiki/Tournaments/LGA), and various [community tournaments](https://osu.ppy.sh/wiki/Community/Bespoke_music), **WangleLine** has developed a style that perfeclty suits the needs of rhythm game players. (Given her background as both an [osu! player](https://osu.ppy.sh/users/7994819) and a game developer, that shouldn't be a surprise.)

Reviewer: ```suggestion
German producer **WangleLine** is as prolific as they get. With 14 albums created to date, plus tracks for the [osu!catch World Cup](https://osu.ppy.sh/wiki/Tournaments/CWC), [Lazer Grand Arena](https://osu.ppy.sh/wiki/Tournaments/LGA), and various [community tournaments](https://osu.ppy.sh/wiki/Community/Bespoke_music), **WangleLine** has developed a style that perfectly suits the needs of rhythm game players. (Given her background as both an [osu! player](https://osu.ppy.sh/users/7994819) and a game developer, that shouldn't be a surprise.)
```
